nuxeo-wopi/src/main/java/org/nuxeo/wopi/jaxrs/FilesEndpoint.java

```diff
@@ -470,15 +470,14 @@ public Object putRelativeFile() {
         map.put(HOST_VIEW_URL, hostViewUrl);
         map.put(HOST_EDIT_URL, hostEditUrl);
         logResponse(OPERATION_PUT_RELATIVE_FILE, OK.getStatusCode(), map);
-        return Response.ok(WOPIMap.of(map)).build();
+        return Response.ok(map).type(MediaType.APPLICATION_JSON).build();
     }
 
     /**
      * Implements the RenameFile operation.
      * <p>
      * See <a href="https://wopi.readthedocs.io/projects/wopirest/en/latest/files/RenameFile.html"></a>.
      */
-    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Why shouldn't/can't we use it?
And in this case, shouldn't we also remove it from `checkFileInfo`?

Reply:
Because it does nothing as it's not a method called by JAX-RS (no @Path, @GET, ...) otherwise, `checkFileInfo` is.